flow/layers/physical_shape_layer.h

```diff
@@ -37,11 +37,18 @@ class PhysicalShapeLayer : public ContainerLayer {
 
 #if defined(OS_FUCHSIA)
   void UpdateScene(SceneUpdateContext& context) override;
+
+  void UpdateSceneChildren(SceneUpdateContext& context);
 #endif  // defined(OS_FUCHSIA)
 
   float total_elevation() const { return total_elevation_; }
 
  private:
+#if defined(OS_FUCHSIA)
+  float local_scenic_elevation_ = 0.0f;
```

Comment:
Actually, #1 is also unnecesary if following one of my suggestions above

Reply:
+1

Reply:
Done.